rust/core-lib/src/view.rs

```diff
@@ -644,6 +644,23 @@ impl View {
         }
     }
 
+    /// Determines the current number of find results and search parameters to send them to
+    /// the frontend.
+    pub fn send_find_status(&self, client: &Client) {
```

Comment:
as above I think I'd prefer if this just returned some `FindStatus` type, which could be sent from the parent context.